src/main/scala/nl/tudelft/fruitarian/observers/ExperimentObserver.scala

```diff
@@ -56,14 +57,14 @@ class ExperimentObserver(handler: TCPHandler, transmissionObserver: Transmission
 
     val avgTimePerRound: Double = timeDiff / rounds
     val avgTimePerRoundCorrected: Double = timeDiff / correctRounds
-    val theoreticalMaxBandwidth: Double = 1000 / avgTimePerRound * 280
-    val actualMaxBandwidth: Double = 1000 / avgTimePerRoundCorrected * 280
-    val prettyTheoreticalMaxBandwidth = (math rint theoreticalMaxBandwidth * 8) / 1000
-    val prettyActualMaxBandwidth = (math rint actualMaxBandwidth * 8) / 1000
+    val theoreticalMaxBandwidth: Double = 1024 / avgTimePerRound * DCnet.MESSAGE_SIZE
+    val actualMaxBandwidth: Double = 1024 / avgTimePerRoundCorrected * DCnet.MESSAGE_SIZE
```

Comment:
```suggestion
    val theoreticalMaxBandwidth: Double = 1000 / avgTimePerRound * DCnet.MESSAGE_SIZE
    val actualMaxBandwidth: Double = 1000 / avgTimePerRoundCorrected * DCnet.MESSAGE_SIZE
```
These lines calculate the rate of bytes per second. The `TimePerRound` is measured in milliseconds, so it should be 1000ms divided by the average time per round in order to get the amount of rounds per second.